Description: Set the default RIPCache to "auto". This makes Ghostscript (pstoraster, pdftoraster) determine the needed amount of memory completely automatically, which makes nearly all files rendering correctly with the CUPS Raster output device. This requires Ghostscript 9.00 or newer or Ghostscript with the appropriate change patched in.  The upstream hardcoded default of 8 MB (default of CUPS and of Ghostscript 8.71 and older) is too small for complex documents.  The RIPCache is also used by the imagetops CUPS filter, which will default to 32 MB with our "auto" setting. It worked already perfectly with the old 8 MB.
Author: Till Kamppeter <till.kamppeter@gmail.com>

Index: cups-1.4.6/scheduler/conf.c
===================================================================
--- cups-1.4.6.orig/scheduler/conf.c	2011-07-14 13:02:44.329772752 +0200
+++ cups-1.4.6/scheduler/conf.c	2011-07-14 13:02:47.689772810 +0200
@@ -542,7 +542,7 @@
 
   cupsdClearString(&DefaultPaperSize);
 
-  cupsdSetString(&RIPCache, "8m");
+  cupsdSetString(&RIPCache, "auto");
 
   cupsdSetString(&TempDir, NULL);
 
Index: cups-1.4.6/scheduler/cupsfilter.c
===================================================================
--- cups-1.4.6.orig/scheduler/cupsfilter.c	2011-07-14 13:02:00.459772006 +0200
+++ cups-1.4.6/scheduler/cupsfilter.c	2011-07-14 13:02:47.689772810 +0200
@@ -1218,7 +1218,7 @@
   else
     set_string(&FontPath, CUPS_FONTPATH);
 
-  set_string(&RIPCache, "8m");
+  set_string(&RIPCache, "auto");
 
   if ((temp = getenv("CUPS_SERVERBIN")) != NULL)
     set_string(&ServerBin, temp);
